src/main/java/org/killbill/billing/plugin/adyen/api/mapping/klarna/PropertyMapper.java

```diff
@@ -100,7 +104,7 @@ public String toString() {
         private Long amountIncludingTax;
 
         // fields included for merchant data
-        private String inventoryService;
+        private String inventoryType;
```

Comment:
check for backward compatibility as we are renaming variable

Reply:
Already taken care.